proxy/src/main/java/com/velocitypowered/proxy/connection/MinecraftConnection.java

```diff
@@ -164,6 +165,10 @@ public EventLoop eventLoop() {
    * @param msg the message to write
    */
   public void write(Object msg) {
+    if (msg instanceof BossBar && protocolVersion.getProtocol() < ProtocolVersion.MINECRAFT_1_9
```

Comment:
Let's not have this check in `MinecraftConnection`. This is an obvious hot-spot and it needs to be free of any big code paths.

Instead, for 1.8, simply skip sending packets to them in the `VelocityBossBar` class.